### PR TITLE
Enhancement/new options panel

### PR DESCRIPTION
**Pull request by Gursaanj**
New option panel to match Figma board

NewOptionPanelGif

### CODE REVIEW

**Comment by Gursaanj** [comment]
[Update UI Panels](https://trello.com/c/8NAONC4S/399-update-ui-panels)

File: Assets/Script/UI/MainMenu/OptionPanelBehaviour.cs [hunk 1]
```diff
@@ -4,131 +4,82 @@
...
-    // When UI element is selected, highligt corresponding text
-    private void HighlightSelectedOption(BaseEventData eventdata)
+    private void OptionsPanelOnUpdate()
     {
-        Vector3 textPosition = m_SFXText.transform.position;
-        if (eventdata.selectedObject.Equals(m_BGMSlider.gameObject))
+        if (PerigonGames.InputManager.IsCancelPressed())
         {
-            textPosition = m_BGMText.transform.position;
+            GameEventMessage.SendEvent("BackButton");
```
Comment (m4rtint) [inline Assets/Script/UI/MainMenu/OptionPanelBehaviour.cs]: TODO - This should be placed inside Stored String When my PR is merged.

File: Assets/Script/UI/VolumeSlider.cs [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    private void CheckDependancies()
+    {
+        Warnings.CheckObject(m_slider, WarningType.MissingChildReference, GetType());
+    }
+
+    private void Awake()
+    {
+        CheckDependancies();
```
Comment (m4rtint) [inline Assets/Script/UI/VolumeSlider.cs]: Typo

File: Assets/Script/UI/VolumeSlider.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        eventTrigger.triggers.Add(m_onSelectTriggerEntry);
+        eventTrigger.triggers.Add(m_onDeselectTriggerEntry);
+    }
+
+    private void CheckDependancies()
+    {
+        Warnings.CheckObject(m_slider, WarningType.MissingChildReference, GetType());
+    }
+
+    private void Awake()
```
Comment (m4rtint) [inline Assets/Script/UI/VolumeSlider.cs]: Put everything in here inside OnEnable

And then OnDisable - Remove AllListeners.

**Comment by m4rtint** [comment]
I'll try to fix the doozy and main menu scene

**Comment by DJSteins** [comment]
Can you make sure that all method comments above the method declarations use the /// <summary> standard?

**Pull request merged**